### DISCUSSION TITLE
Project Server URL disappear

### DISCUSSION THREAD

**Discussion by fxzer** [nuxt/nuxt#21603] [General]
Why do the URL disappear after the project starts, and sometimes the rolling Terminal cannot roll up to see them.
![image](https://github.com/nuxt/nuxt/assets/68853957/9846f3db-4b11-42cd-a8a8-def810d6f957)![image](https://github.com/nuxt/nuxt/assets/68853957/f0595ef8-7fac-45fd-b300-19d08b78812e)

I hope to see those existences all the time，No need to restart the project every time to see！

**Comment by warflash** [comment]
It scrolls up so you can see actual logs being printed which imo does make sense 🤔

**Comment by fxzer** [comment]
bu i cant scrollup
---- Replied Message ----
| From | ***@***.***> |
| Date | 06/17/2023 10:19 |
| To | ***@***.***> |
| Cc | fxzer ***@***.***>,
Author ***@***.***> |
| Subject | Re: [nuxt/nuxt] Project Server URL disappear (Discussion #21603) |

It scrolls up so you can see actual logs being printed which imo does make sense 🤔
